src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -93,8 +93,16 @@ public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager
         [HttpPost]
         [Route("admin/host/ping")]
         [ResponseCache(NoStore = true, Location = ResponseCacheLocation.None)]
-        public IActionResult Ping()
+        public IActionResult Ping([FromServices] IScriptHostManager scriptHostManager)
         {
+            var hostStatus = new
+            {
+                hostState = scriptHostManager.State.ToString()
+            };
+
+            string message = $"Ping Status: {JsonConvert.SerializeObject(hostStatus, Formatting.Indented)}";
+            _logger.Log(LogLevel.Information, new EventId(0, "PingStatus"), message);
```

Comment:
Let's make this Verbose, not Info.

Reply:
Also, would be good if the json blob is in the Details column, and Summary was "Host status on ping". That way Kusto json parse operators can be used to parse/query over it easily for detectors.